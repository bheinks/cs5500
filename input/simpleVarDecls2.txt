(*
	Declarations of simple types with more than one
      variable per declaration
*)

program p;

var x : integer;
    y,z : boolean;
    a, b, c, d : char;

begin
  write(5)
end.

