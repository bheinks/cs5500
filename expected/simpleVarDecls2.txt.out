

>>> Entering new scope...

+++ Adding p to symbol table with type PROGRAM

+++ Adding x to symbol table with type INTEGER

+++ Adding y to symbol table with type BOOLEAN

+++ Adding z to symbol table with type BOOLEAN

+++ Adding a to symbol table with type CHAR

+++ Adding b to symbol table with type CHAR

+++ Adding c to symbol table with type CHAR

+++ Adding d to symbol table with type CHAR

<<< Exiting scope...

---- Completed parsing ----
